spec/abstractops.html

```diff
@@ -1027,32 +1027,26 @@ <h1>ISO 8601 grammar</h1>
           `.`
           `_`
 
-      TZChar :
+      TZChar[Legacy] :
           Alpha
           `.`
           `-`
           `_`
+          [+Legacy] `+`
+          [+Legacy] DecimalDigit
 
-      TimeZoneIANANameComponent :
-          TZLeadingChar TZChar? TZChar? TZChar? TZChar? TZChar? TZChar? TZChar? TZChar? TZChar? TZChar? TZChar? TZChar? TZChar? but not one of `.` or `..`
+      TimeZoneIANANameComponent[Legacy] :
+          TZLeadingChar TZChar[?Legacy]? TZChar[?Legacy]? TZChar[?Legacy]? TZChar[?Legacy]? TZChar[?Legacy]? TZChar[?Legacy]? TZChar[?Legacy]? TZChar[?Legacy]? TZChar[?Legacy]? TZChar[?Legacy]? TZChar[?Legacy]? TZChar[?Legacy]? TZChar[?Legacy]? but not one of `.` or `..`
```

Comment:
Another possibility:
```suggestion
      TimeZoneIANANameComponent[Legacy] :
          [~Legacy] TZLeadingChar TZChar? TZChar? TZChar? TZChar? TZChar? TZChar? TZChar? TZChar? TZChar? TZChar? TZChar? TZChar? TZChar? but not one of `.` or `..`
          [+Legacy] TZLeadingChar TZChar[+Legacy]? TZChar[+Legacy]? TZChar[+Legacy]? TZChar[+Legacy]? TZChar[+Legacy]? TZChar[+Legacy]? TZChar[+Legacy]? TZChar[+Legacy]? TZChar[+Legacy]? TZChar[+Legacy]? TZChar[+Legacy]? TZChar[+Legacy]? TZChar[+Legacy]? but not one of `.` or `..`
```
I'm not sure which is more readable.

Reply:
According to https://github.com/tc39/ecma262/pull/2781#discussion_r974904811 I've learned that we should probably spell out the 14 different possibilities explicitly, so keeping `[?Legacy]` in order for it not to be 28 seems more readable... I'll make that change.